### DISCUSSION TITLE
Aggregated TODO list

### DISCUSSION THREAD

**Discussion by parashardhapola** [parashardhapola/scarf#46] [Ideas]
Using this issue to track all the feature requests. I have tried to loosely categorize them as well

#### New functionality

- [x] Data 'bulkification': The idea here is to merge the data from cells within the same clusters. This can then have two downstream usages: 1) bulk data can be exported out for use in external software like GSEA and 2) mapping speed up mapping only to bulked values. (Solved here: https://github.com/parashardhapola/scarf/commit/aad885a66651a1acb99154a4b85e48770c1a7a39)

#### Improved support

- [ ] Extend meld_assay to support enhancers and TF motifs

#### Visualization
- [ ] Add Beadplot
- [ ] Add hullplots

#### Support for formats and dataset management

- [x] Anndata to Zarr convertor (Solved here: https://github.com/parashardhapola/scarf/commit/d41c3df3341baeebbb21828c653143dba7d735d4 and https://github.com/parashardhapola/scarf/commit/04f20e224394b466875e92cb0327a233653d8545)

- [x] Zarr to Anndata object
- [x] Merge two or more Zarr files

**Comment by razofz** [comment]
"Merge two or more Zarr files" done, considering class `ZarrMerge` exists?
